paymentsheet-example/src/androidTestDebug/java/com/stripe/android/utils/PaymentIntentFactory.kt

```diff
@@ -0,0 +1,38 @@
+package com.stripe.android.utils
+
+import com.stripe.android.model.CardBrand
+import com.stripe.android.model.PaymentIntent
+import com.stripe.android.model.PaymentMethod
+import com.stripe.android.model.StripeIntent
+
+internal object PaymentIntentFactory {
```

Comment:
Copied over from `paymentsheet`. Eventually, we should move these mock factories to a separate module.